dev-docs/design/how-lit-works.md

```diff
@@ -187,18 +187,18 @@ Because this step does not use any provided values (and because templates are in
 
 ### 2.3. Create a lit-html `Template`
 
-The `Template` class does most of the heavy-lifting for preparing a template. It walks the tree of nodes in a `<template>` element finding the markers inserted in step 2.1. If a marker is found, either in an attribute, as a marker comment node, or in text content, the depth-first index of the node is recorded, along with the type of expression (`'text'` or `'attribute'`) and the name of the attribute. The metadata containing the depth-first index of the node and type of expression is called a `TemplatePart` in the source.
+The `Template` class does most of the heavy-lifting for preparing a template. It walks the tree of nodes in a `<template>` element finding the markers inserted in step 2.1. If a marker is found, either in an attribute, as a marker comment node, or in text content, the depth-first index of the node is recorded, along with the type of expression (`'text'` or `'attribute'`) and the name of the attribute. The metadata containing the depth-first index of the node and type of expression is called a `TemplatePart` in the source. A `TemplatePart` is not the same as the `Parts` mentioned [in the Parts section](#parts), but is the inert definition for what `Part` should be created later during the [Create Parts](#33-create-parts) phase.
```

Comment:
I like the extra description of `TemplatePart` but the `[Create Parts](#33-create-parts)s` no longer exists and should point to `3.2 Clone the template and instantiate Parts`.